examples/users.ndjson

```diff
@@ -6,5 +6,5 @@
 { "name": "sixth",   "birthday": "1962-01-01", "ip": "6.1.1.1" }
 { "name": "seventh", "birthday": "1951-01-01", "ip": "7.1.1.1" }
 { "name": "eighth",  "birthday": "1940-01-01", "ip": "8.1.1.1" }
-{ "name": "nineth",  "birthday": "1939-01-01", "ip": "9.1.1.1" }
+{ "name": "ninth",  "birthday": "1939-01-01", "ip": "9.1.1.1" }
```

Comment:
Oops, can you add additional whitespace to align this line? Thank you!

Reply:
Also just noticed the same error also exists in https://github.com/clue/reactphp-csv (https://github.com/clue/reactphp-csv/pull/32), https://github.com/clue/reactphp-ndjson and https://github.com/clue/reactphp-tsv.

Reply:
Thank you for your input. I have already noticed the typos in https://github.com/clue/reactphp-csv/pull/32. I will soon work with @SimonFrings to update the test suite, so hopefully it will pass successfully.

I will also keep the other two in mind as I make progress on fixing the typos in the projects. I will include a link to your comment here in my future response.

Reply:
also fixed the whitespace - oops, thank you 😉